src/Centreon/Infrastructure/HostConfiguration/Repository/HostSeverityRepositoryRDB.php

```diff
@@ -0,0 +1,126 @@
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+declare(strict_types=1);
+
+namespace Centreon\Infrastructure\HostConfiguration\Repository;
+
+use Centreon\Domain\HostConfiguration\Interfaces\HostSeverityReadRepositoryInterface;
+use Centreon\Domain\RequestParameters\RequestParameters;
+use Centreon\Infrastructure\DatabaseConnection;
+use Centreon\Infrastructure\HostConfiguration\Repository\Model\HostSeverityFactoryRdb;
+use Centreon\Infrastructure\Repository\AbstractRepositoryDRB;
+use Centreon\Infrastructure\RequestParameters\Interfaces\NormalizerInterface;
+use Centreon\Infrastructure\RequestParameters\SqlRequestParametersTranslator;
+
+/**
+ * This class is designed to represent the MariaDb repository to manage host severity.
+ *
+ * @package Centreon\Infrastructure\HostConfiguration\Repository
+ */
+class HostSeverityRepositoryRDB extends AbstractRepositoryDRB implements HostSeverityReadRepositoryInterface
+{
+    /**
+     * @var SqlRequestParametersTranslator
+     */
+    private $sqlRequestTranslator;
+
+    public function __construct(DatabaseConnection $db, SqlRequestParametersTranslator $sqlRequestTranslator)
+    {
+        $this->db = $db;
+        $this->sqlRequestTranslator = $sqlRequestTranslator;
+        $this->sqlRequestTranslator
+            ->getRequestParameters()
+            ->setConcordanceStrictMode(RequestParameters::CONCORDANCE_MODE_STRICT);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function findHostSeverities(): array
+    {
+        $this->sqlRequestTranslator->setConcordanceArray(
+            [
+                'id' => 'hc_id',
+                'name' => 'hc_name',
+                'alias' => 'hc_alias',
+                'level' => 'level',
+                'is_activated' => 'hc_activate',
+            ]
+        );
+        $this->sqlRequestTranslator->addNormalizer(
+            'is_activated',
+            new class implements NormalizerInterface
+            {
+                public function normalize($valueToNormalize)
+                {
+                    if (is_bool($valueToNormalize)) {
+                        return ($valueToNormalize === true) ? '1' : '0';
+                    }
+                    return $valueToNormalize;
+                }
+            }
+        );
+        $request = $this->translateDbName(
+            'SELECT SQL_CALC_FOUND_ROWS hc.*, icon.img_id AS img_id, icon.img_name AS img_name,
+                CONCAT(iconD.dir_name,\'/\',icon.img_path) AS img_path, icon.img_comment AS img_comment
```

Comment:
You used the DIRECTORY_SEPARATOR here : 
https://github.com/centreon/centreon/pull/9371/files#diff-0b917d5735d4715bfbe78685b4a9d705f9aefbe58c3b7e4d7e61d86e0b3442d4R101
Meaning that / or \ can be used
```suggestion
                CONCAT(iconD.dir_name,\'' . DIRECTORY_SEPARATOR . '\',icon.img_path) AS img_path, icon.img_comment AS img_comment
```

Reply:
path is for web, not for system